### PR TITLE
#1009 avoid including non used createDecimalFormat helper method

### PR DESCRIPTION
**Pull request by sjaakd**

### CODE REVIEW

File: processor/src/test/java/org/mapstruct/ap/test/conversion/bignumbers/BigNumbersConversionTest.java [hunk 1]
```diff
@@ -188,4 +190,22 @@ public void shouldApplyReverseBigDecimalConversions() {
...
+    @IssueKey("1009")
+    @WithClasses({ BigIntegerSource.class, BigIntegerTarget.class, BigIntegerMapper.class })
+    public void shouldNotGenerateCreateDecimalFormatMethod() {
+
+        Class clz = BigIntegerMapper.INSTANCE.getClass();
+        try {
+            Method method = clz.getDeclaredMethod( "createDecimalFormat", String.class );
+            assertThat( method ).isNull();
+        }
+        catch ( NoSuchMethodException ex ) {
```
Comment (filiphr) [inline processor/src/test/java/org/mapstruct/ap/test/conversion/bignumbers/BigNumbersConversionTest.java]: A bit nitpicking :smile:, but you can use our `GeneratedSource` JUnit Rule for this.  See the usages of it to see how you can do it

Comment (sjaakd) [inline processor/src/test/java/org/mapstruct/ap/test/conversion/bignumbers/BigNumbersConversionTest.java]: @filiphr : good idea.. There's more stuff than I realized :smile:

**Comment by sjaakd** [comment]
thanks @filiphr for reviewing

**Comment by gunnarmorling** [comment]
Nice, thanks!

**Pull request merged**